spec/models/dfe_sign_in/user_data_importer_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe DfeSignIn::UserDataImporter, type: :model do
+  before { stub_dfe_sign_in_user_list_request }
+
+  it "imports all user data" do
+    DfeSignIn::UserDataImporter.new.run
+
+    expect(DfeSignIn::User.count).to eq(3)
```

Comment:
What about verifying the actual values of the user records are being correctly set? Seems like a whole heap of behaviour isn't being tested here.